app/controllers/invite_controller.rb

```diff
@@ -20,6 +20,15 @@ def submit
     first_name = params[:first_name]
     last_name = params[:last_name]
 
+    if ENV["RESTRICTED_DOMAIN"]
+      if email.split("@").last != ENV["RESTRICTED_DOMAIN"]
+        @message = "Sorry! Early access is currently restricted to people within the #{ENV["RESTRICTED_DOMAIN"]} domain.  We will be opening it up soon, though!"
```

Comment:
I'd remove the `  We will be opening it up soon, though!` as it might not be correct for everyone.